.github/workflows/ci.yml

```
@@ -15,18 +15,19 @@ jobs:
       AIRBRAKE_KEY: longvaluefullofnumbersandlettersinlowercase
       PORT: 3000
       # External services - they don't exist here but we have code that depends on these values being present
-      EA_ADDRESS_FACADE_URL: http://localhost:8009
       CHARGING_MODULE_URL: http://localhost:8020
-      SERVICE_FOREGROUND_URL: http://localhost:8001
-      SERVICE_BACKGROUND_URL: http://localhost:8012
-      REPORTING_URL: http://localhost:8011
-      IMPORT_URL: http://localhost:8007
-      TACTICAL_CRM_URL: http://localhost:8002
+      EA_ADDRESS_FACADE_URL: http://localhost:8009
       EXTERNAL_UI_URL: http://localhost:8000
+      HTML_TO_PDF_URL: http://localhost:8040
```

I'm going to ask to rename this `GOTENBERG_URL` to match the others that are named based on service name rather than purpose. This will then also apply to the other env vars and config.

But I appreciate the ordering change! 😁 

```suggestion
      GOTENBERG_URL: http://localhost:8040
```